index.js

```diff
@@ -12,11 +12,17 @@ const { log } = console;
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 
-app.use('/api/v1', routes);
+app.get('/', (req, res) => {
+  res.status(200).json({
+    message: 'Hello Books Deferral'
+  });
+});
+
+app.use('/api/v1/', routes);
```

Comment:
Thanks for adding the base API route back to the logic. I think this looks good to go! Great job once again!

Reply:
Good job @abdulfatai360 .  I think all looks fine.